default.nix

```diff
@@ -348,7 +362,10 @@ in rec {
 
       linuxExeConfigurable = linuxExe;
       linuxExe = linuxExe dummyVersion;
-      exe = serverOn mainProjectOut dummyVersion;
+      exe = serverOn mainProjectOut null dummyVersion;
+      inherit vm;
+      ova = args@{ hostName, adminEmail, routeHost, enableHttps, version, module ? (_: _: {}) }:
+        (vm args).config.system.build.virtualBoxOVA;
```

Comment:
I think we should put these behind an `__unstable__` attr for now until we actually officially use them inside obelisk. Would that mess up your ability to use it?

Reply:
Shouldn't hinder me.
Do you mean the same [`__unstable__` where `profiledObRun` is](https://github.com/obsidiansystems/obelisk/pull/721/files#diff-5712e736e0de6ba170577f8472c398e9R335)?

Reply:
Yes